exec/jdbc-all/pom.xml

```diff
@@ -576,7 +576,7 @@
                           This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
 
                         </message>
-                        <maxsize>37500000</maxsize>
+                        <maxsize>38600000</maxsize>
```

Comment:
are these due to both `jetty` and `jersey` dependencies being pulled in ? We should run mvn dependency:tree command with and without the PR to see.

Reply:
I forgot to check jdbc-all exclusions.
So `jetty` was fully excluded before updates. And almost all `Glassfish Jersey` dependencies were excluded. But I had added new one without exclusion - `org.glassfish.jersey.inject:jersey-hk2`.
For now it is excluded and jar file size can be untouched.